lib/services/http_service.rb

```diff
@@ -19,22 +19,33 @@ def initialize(host)
 
     # @param endpoint     - e.g. '/context'
     # @param query_params - e.g. {a:1,b:2}
-    def get(endpoint = '/', query_params = nil)
-      make_request :get, endpoint, query_params
+    DEFAULT_GET_PARAMS = { endpoint: '/', query_params: nil, body: nil, headers: {} }
```

Comment:
i've never seen constants put just above the methods.  i understand the reasoning, just never seen it :)